appinventor/buildserver/src/com/google/appinventor/buildserver/Main.java

```diff
@@ -48,21 +48,25 @@
     String dexCacheDir = null;
 
     @Option(name = "--includeDangerousPermissions",
-        usage = "Add extra features not allowed in the Google Play store.")
+            usage = "Add extra features not allowed in the Google Play store.")
```

Comment:
Continuing indentation should remain 4 spaces. See other lines in this file as well.

Reply:
Fixed. IntelliJ again.